src/src/gameplay/cards/CardBlue.gd

```diff
@@ -4,6 +4,8 @@ class_name CardBlue
 
 export(float) var Duration = 5.0
 export(Color, RGB) var col = Color(0, 0, 1)
+export(bool) var reveal_enemies = false
```

Comment:
I see the problem with these now, needing this var in cards that don't use it is not nice. If this starts being a problem maybe we can do either:
 - EnemyManager listens to the event and keeps track of the current state
 - EnemyManager does `if "reveal_enemies" in card_manager.current_card: determine_visibility`
 For now thought I would just leaving like this